app/api/routes.py

```diff
@@ -0,0 +1,57 @@
+from flask import jsonify, request, current_app, url_for
+from . import api
+from ..models import Plant
+from flask_sqlalchemy import SQLAlchemy
+from app import db
+
+
+@api.route('/')
+def endpoint():
+    return "I need to go take a shower so I can't tell if I'm crying or not."
+
+@api.route('/plants/<int:id>')
+def get_plant(id):
+    plant = Plant.query.filter_by(id=id).first()
+    result = {
+                'id': plant.id,
+                'name': plant.name,
+                'plant_type': plant.plant_type
+                }
+    response = jsonify(result)
+    response.status_code = 200
+    return response
+
+@api.route('/plants/')
+def all_plants():
+    plants = Plant.get_all()
+    results = []
+
+    for plant in plants:
+        obj = {
+            'id': plant.id,
+            'name': plant.name
+        }
+        results.append(obj)
+    response = jsonify(results)
+    response.status_code = 200
+    return response
+
+@api.route('/plants/meet')
+def random_plant():
+    from random import randint
+    id = randint(1, 24)
```

Comment:
We should find a way to get the number of records in the table to generate a random number off of. I think we could do this with a get_all().count or something similar

Reply:
LOVE IT.